packages/create-pantheon-decoupled-kit/src/utils/runESLint.ts

```diff
@@ -21,12 +31,24 @@ export const runESLint = (
 	} else {
 		command = getPkgManager.name;
 	}
-	try {
-		execSync(`${command} lint`, { cwd: answers.outDir, stdio: 'inherit' });
-	} catch (error) {
-		if (error instanceof Error) {
-			throw error;
+
+	const filePath = path.resolve(`${answers.outDir}/package.json`);
+	const script = JSON.parse(fs.readFileSync(filePath, 'utf8')) as Script;
+	if (script.scripts.lint) {
+		try {
+			execSync(`${command} lint`, { cwd: answers.outDir, stdio: 'inherit' });
```

Comment:
Unsure this does anything without providing file extensions or paths to lint. Need to do some more validation here but I think we at least need `npm run lint .` to tell the cli where/what to lint.